Sources/PostgresNIO/New/PSQLTask.swift

```diff
@@ -31,16 +33,18 @@ enum PSQLTask {
     }
 }
 
+// FIXME: Either rename all these `ExtendedQuery`s to just like `Query` or pull out `simpleQuery`
 final class ExtendedQueryContext {
     enum Query {
         case unnamed(PostgresQuery, EventLoopPromise<PSQLRowStream>)
         case executeStatement(PSQLExecuteStatement, EventLoopPromise<PSQLRowStream>)
         case prepareStatement(name: String, query: String, bindingDataTypes: [PostgresDataType], EventLoopPromise<RowDescription?>)
+        case simpleQuery(String, EventLoopPromise<PSQLRowStream>)
     }
```

Comment:
`simpleQuery`  being here doesn't quite make sense.

As i see, our options are:
* Rename all these `ExtendedQuery` types to just `Query`. (still doesn't make great sense)
* Create a new type and all, and possibly a new state machine for `SimpleQuery`.

@fabianfett WDYT?